### PR TITLE
<Fixes#967> Fix button names in download notification 

### LINKED ISSUE #967: fix button names in download notification
Instead of "Play", we should Probably have the string "Resume"

**Steps to reproduce the behavior:**
1. Download a zim file
2. Pause the download
3. Open notifications

**Screenshots**

screenshot_20190210-190737

**Environment**
- Version of Kiwix Android : 2.4 Build 55
- Device : Samsung S6
- OS version : Android 7.0

### PR DESCRIPTION
**Pull request by iadeelzafar**
### Changes:

* add Resumes string value
* add Translation for other languages
* add this label to the download notification label instead of PLAY.

### CODE REVIEW

**Comment by iadeelzafar** [comment]
Hi @abdulwd , There's something I need your help with.  Day before yesterday I pulled a branch from upstream master that soloturn had related to Kotlin to work on it. After that, yesterday and also today, I pulled master branch from upstream into my master. Now, in order to work on this issue, I made a new branch from my local master->resumee. Now, after solving this issue when I've committed it to my origin and i'm making this Pull request, I see all those commits. How can I squash these because these are not exactly the last commits in sequence. When I try squashing last 6, I see that there are even commits that were merged yesterday by mhutti1.

**Comment by abdulwd** [comment]
You must have used the `git pull` command when fetching soloturn's branch and you must have been on the master. `git pull` command is the combination of `git fetch` and `git merge` so it fetched the commits made by Soloturn and merged them in the master branch. So you should only pull the changes from upstream to the master branch using `git pull` to keep your code synced with this repo. If you want to work on someone's branch you should fetch his/her branch using `git fetch upstream <branch-name>` and then check out that branch.

**Comment by iadeelzafar** [comment]
@abdulwd Got it. It means when pulling some other branch you want to temp work on, before pulling, you must checkout a new branch specifically for that or just use fetch. So, how can I get out of this now?

**Comment by iadeelzafar** [comment]
@abdulwd Since the last 4 hours, I was trying to resolve this, I was searching and trying different git commands and I ended up deleting build.gradle files mistakenly. I pulled the project again from my origin but still it can't  find the build gradle neither pulling from upstream is working. I think I'll have to re fork and re clone the project. :(

File: app/src/main/res/values-af/strings.xml [hunk 1]
```diff
@@ -31,4 +31,5 @@
   <string name="pref_language_chooser">Kies \'n taal</string>
   <string name="menu_bookmarks_list">Boekmerke</string>
   <string name="zim_manager">Biblioteek</string>
```
Comment (mhutti1) [inline app/src/main/res/values-af/strings.xml]: These are generated from translatewiki.net. Only edit the english string.

**Comment by iadeelzafar** [comment]
@abdulwd @mhutti1 I re-cloned the project and this what I'm getting on importing it.
Which option to go for?
image

image

**Comment by mhutti1** [comment]
I think you should be able to just open it. No need to import. If you are importing then I might try Android gradle not gradle?

**Comment by iadeelzafar** [comment]
@mhutti1 I've tried Android Gradle as well. I've tried several things since morning. Nothing is working for me. I think maybe it's happening because of the shift to the build.gradlle.kts. Can someone else try recloning the project and then opening it to see if it works or not. I'm stuck. @soloturn could you help me out?

**Comment by abdulwd** [comment]
@iadeelzafar I have cloned the project again and was facing the same issue as you are. You can follow these steps and check if the issue is resolved:
- Create a `local.properties` file in the project directory and add the location of the SDK according to this [answer](https://stackoverflow.com/a/48155800/7792129).
- Delete the `.idea` folder in the project directory. It is a hidden folder so you may have to change the settings of your file explorer to view it.
- Import the project using **Open an existing Android Studio project** and a dialog will pop up select these settings:
image

**Comment by iadeelzafar** [comment]
@abdulwd I can't thank you enough. I tried this sdk solution earlier but it didn't worked but now after deleting the .idea folder, it has finally worked.

**Pull request closed without merging**